### ISSUE TITLE
Converting video to opus audio file removes the first second of the output song file

### ISSUE THREAD

**Issue by STRRRRR** [shaked6540/YoutubePlaylistDownloader#26]
I used the ''convert videos to opus'' option in the audio part of the configuration and got in result downloaded an audio file with 1 second skipped. If you need me to be more specific please let me know.

**Comment by shaked6540** [comment]
Yeah, I'll need some more specific information: 
1. the video you're trying to download
2. the full settings you used (you can either screenshot the settings tab, or send me the contents of the app data directory (`%appdata%\Youtube Playlist Downloader`)

**Comment by STRRRRR** [comment]
I appreciate the quick response and I also apologize for not using screenshots to make it easier to understand the issue. 
Anyway, I've been trying to extract the audio from this music video https://www.youtube.com/watch?v=1Ck4HbYdzS4 . I downloaded it 2 times with your program using 2 different configurations:

1. Downloading the video as it is, without converting it into any format https://imgur.com/iCBXUrJ
2. Downloading and converting it to opus https://imgur.com/1Nni5WJ

So after playing both files using VLC media player I noticed the opus audio had approximately the first second of the song removed compared to the other one. 

Hopefully this is only an issue on my end

**Comment by shaked6540** [comment]
I've found the problem. something with the library I'm using to insert tags to audio files cuts the very beginning of the converted file, seems like it happens with any opus file, maybe the library doesn't like opus files?

I will try to investigate this even further tomorrow, and if I can't find a solution I'll add an option to disable tagging audio files and release an update

I also found another bug when investigating this, the "set the bitrate" checkbox doesn't work, it will always set the bitrate that is in the textbox near it unless its empty, so double thanks :)

**Comment by STRRRRR** [comment]
Looking forward the fix; this is the best program I found so far to download music.

**Comment by shaked6540** [comment]
@STRRRRR hey, just wanted to let you know I've released an update to fix this issue. Just open the program and it will ask you if you want to update.

Opus files will not be tagged anymore because of the library bug, and I added an option to choose whether or not to tag other file formats.

I couldn't fix the opus file tagging problem because it uses an external library that I can't really fix, sorry for that. 

Have fun downloading!
